website/src/css/imports/docs.css

```diff
@@ -144,6 +144,16 @@
       padding: initial;
       white-space: inherit;
     }
+
+    pre {
+      margin: 30px 0 !important;
+    }
+
+    @media (min-width: $xlarge) {
+      pre {
+        margin-right: -100px !important;
+      }
+    }
```

Comment:
Override prism theme code block margins, extend to the right a bit when the screen is large enough.